src/usb_reset/src/owr_ros_components.cpp

```diff
@@ -0,0 +1,13 @@
+/*
+ * Date Started: 25/07/16
+ * Original Author: Harry J.E Day
+ * Editors:
+ * Purpose: C++ code representation of the QML package
+ * This code is released under the MIT License. Copyright BLUEsat UNSW, 2017
+ */
```

Comment:
please fix comment -> also please use a new type here as this name already exists